src/index.test.ts

```diff
@@ -889,3 +889,514 @@ test('using custom levels does not leak into types or runtypes of other loggers'
   expect(infoLog.msg).toBe('info');
   expect(infoLog.name).toBe('my-app');
 });
+
+describe('eeeoh', () => {
+  const { destination, stdoutMock } = rootModule.createDestination({
+    mock: true,
+  });
+
+  afterEach(stdoutMock.clear);
+
+  test('static root config', () => {
+    const logger = createLogger(
+      {
+        eeeoh: { datadog: 'tin' },
+        service: 'deployment-service-name',
+      },
+      destination,
+    );
+
+    logger.info('tin from root option');
+
+    logger.info(
+      {
+        eeeoh: { datadog: 'silver' },
+      },
+      'silver from inline object',
+    );
+
+    logger
+      .child({
+        eeeoh: { datadog: 'bronze' },
+      })
+      .info(
+        {
+          eeeoh: { datadog: 'silver' },
+        },
+        'silver from inline object over child bindings',
+      );
+
+    logger.info(
+      {
+        eeeoh: { datadog: ['bronze', { info: 'silver' }] },
```

Comment:
I get that this is likely reusing an interface for the global config but would anybody realistically provide a level specific override in a logger method?